c9metadata-project.settings

```diff
@@ -0,0 +1 @@
+{"filter":false,"title":"project.settings","tooltip":"/.c9/project.settings","undoManager":{"mark":1,"position":1,"stack":[[{"start":{"row":0,"column":0},"end":{"row":37,"column":1},"action":"remove","lines":["{","    \"build\": {","        \"@saveall\": true","    },","    \"instance\": {","        \"@automaticShutdown\": \"30\"","    },","    \"language\": {","        \"tern_defs\": {","            \"json()\": {","                \"browser\": {","                    \"enabled\": true","                },","                \"ecma5\": {","                    \"enabled\": true","                },","                \"jQuery\": {","                    \"enabled\": true","                }","            }","        }","    },","    \"preview\": {","        \"@url\": \"$C9_HOSTNAME:8081/spring-nanotrader-web/\"","    },","    \"python\": {","        \"@path\": \"/usr/local/lib/python3.4/dist-packages:/usr/local/lib/python3.5/dist-packages\"","    },","    \"run\": {","        \"configs\": {","            \"@inited\": true,","            \"json()\": {}","        }","    },","    \"welcome\": {","        \"@first\": true","    }","}"],"id":9},{"start":{"row":0,"column":0},"end":{"row":36,"column":5},"action":"insert","lines":["{","    \"build\": {","        \"@saveall\": true","    },","    \"instance\": {","        \"@automaticShutdown\": \"30\"","    },","    \"language\": {","        \"tern_defs\": {","            \"json()\": {","                \"browser\": {","                    \"enabled\": true","                },","                \"ecma5\": {","                    \"enabled\": true","                },","                \"jQuery\": {","                    \"enabled\": true","                }","            }","        }","    },","    \"preview\": {","        \"@url\": \"$C9_HOSTNAME:8081/spring-nanotrader-web/\"","    },","    \"python\": {","        \"@path\": \"/usr/local/lib/python3.4/dist-packages:/usr/local/lib/python3.5/dist-packages\"","    },","    \"run\": {","        \"configs\": {","            \"@inited\": true,","            \"json()\": {}","        }","    },","    \"welcome\": {","        \"@first\": true","    }"]}],[{"start":{"row":36,"column":5},"end":{"row":37,"column":0},"action":"insert","lines":["",""],"id":10},{"start":{"row":37,"column":0},"end":{"row":37,"column":4},"action":"insert","lines":["    "]},{"start":{"row":37,"column":4},"end":{"row":37,"column":5},"action":"insert","lines":["}"]},{"start":{"row":37,"column":0},"end":{"row":37,"column":4},"action":"remove","lines":["    "]}]]},"ace":{"folds":[],"customSyntax":"javascript","scrolltop":0,"scrollleft":0,"selection":{"start":{"row":26,"column":46},"end":{"row":26,"column":46},"isBackwards":false},"options":{"guessTabSize":true,"useWrapMode":false,"wrapToView":true},"firstLineState":0},"timestamp":1599575203967,"hash":"0b52b3ce8284d656eba3c4399aaad62e9bb83cb3"}
```

Comment:
wow you weren't kidding about this, lol.

I'll try out this fix today.  A less "hacky" approach might include dynamically building this file and using real values for things like `timestamp` and `hash`.  I can also work on this today if this PR ends up working for us